src/view.cpp

```diff
@@ -374,9 +374,9 @@ nonstd::optional<std::uint8_t> View::open_in_browser(const std::string& url,
 		pop_current_formaction();
 		return ret;
 	} else {
-		set_status(strprintf::fmt(_("Running browser: %s"), cmdline));
+		status_line.show_message(strprintf::fmt(_("Running browser: %s"), cmdline));
```

Comment:
I think this should use `show_message_until_finished()`, which automatically removes the message at the end of the block. That's better than calling `show_message("")` because there might be additional messages which can then take over the current one.